www/js/control/ProfileSettings.tsx

```diff
@@ -225,8 +224,6 @@ const ProfileSettings = () => {
       newNotificationSettings.prefReminderTime = m.toFormat('t');
       newNotificationSettings.prefReminderTimeOnLoad = prefs.reminder_time_of_day;
       newNotificationSettings.scheduledNotifs = scheduledNotifs;
-
-      updatePrefReminderTime(false);
```

Comment:
I do think this could have been unintentional. That commit was not supposed to change the behavior of the code

`updatePrefReminderTime` is called when the datepicker is changed. But I think it also needs to be called to set the time pref on the initial load

TODO investigate, check tests, add back if needed